opi/__init__.py

```diff
@@ -6,7 +6,7 @@
 import configparser
 
 import requests
-import lxml.etree
+from lxml import etree
 
```

Comment:
I would prefer the current version as it makes more clear, what is done via which module.

Reply:
Sure, will change that.